src/Nncase.Core/PatternMatch/PatternUtility.cs

```diff
@@ -62,11 +65,11 @@ public static CallPattern IsWildcardCall<T>(string callName, string opName, stri
     public static CallPattern IsWildcardCall<T>(string callName, string opName, Pattern inputPattern)
         where T : Op
         =>
-        IsCall(callName, IsOp<T>(opName, _ => true), GenerateParameters(callName, inputPattern))
-            with
-        {
-            TypePattern = IsType(x => !(x is InvalidType)),
-        };
+            IsCall(callName, IsOp<T>(opName, _ => true), GenerateParameters(callName, inputPattern))
+                with
+            {
+                TypePattern = IsType(x => !(x is InvalidType)),
+            };
 
     public static CallPattern IsWildcardCall<T>(string callName, string opName, Pattern lhsPattern, Pattern rhsPattern)
```

Comment:
Can extract common method `public static CallPattern IsWildcardCall<T>(string callName, string opName, params Pattern[] argPatterns)`

Reply:
This part of the code is just formatted and will be refactored in the new pr.